cli/cmd/initnodes.go

```diff
@@ -442,6 +435,26 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 	return nil
 }
 
+func maybeDownloadSnapshots(ctx context.Context, cfg InitConfig) error {
+	if !cfg.NodeSnapshot {
+		return nil
+	}
+
+	g, ctx := errgroup.WithContext(ctx)
+
+	// Start parallel downloads.
+	g.Go(func() error {
+		return downloadSnapshot(ctx, cfg.Network, cfg.Home, gethClientName)
+	})
+
+	g.Go(func() error {
+		return downloadSnapshot(ctx, cfg.Network, cfg.Home, haloClientName)
+	})
+
+	// Wait for all downloads to complete.
+	return errors.Wrap(g.Wait(), "parallel download snapshots")
```

Comment:
you risk wrapping nil error.... which panics.... 

nit: this should have paniced during happy flow testing....

Reply:
Wanted to spare an additional error check and make the linter happy at the same time, but you are right and it still needs the check. I will fix it.